content/5_Plat/Sweep_Line.mdx

```diff
@@ -173,9 +173,81 @@ For each point, we recalculate $\delta = \min(\delta, \delta_p)$, and update our
 
 <FocusProblem problem="manSam" />
 
-(KACTL code)
+The key observation is that although there are many points, they are spread across a small surface i.e. $0 \le x, y \le 1000$. So instead of using  Kruskal's or Prim's
```

Comment:
```suggestion
The key observation is that although there are many points, they are spread across a small surface i.e. $0 \leq x, y \le 1000$. So instead of using Kruskal's or Prim's
```

Reply:
Done